src/benchmark/options.ts

```diff
@@ -0,0 +1,40 @@
+import {BenchmarkOpts} from "../types.js";
+
+export const defaultBenchmarkOptions: Required<BenchmarkOpts> = {
+  minRuns: 1,
+  maxRuns: Infinity,
+  minMs: 100,
+  maxMs: Infinity,
+  maxWarmUpRuns: 1000,
+  maxWarmUpMs: 500,
+  convergeFactor: 0.5 / 100, // 0.5%
+  runsFactor: 1,
+  yieldEventLoopAfterEach: false,
+  timeoutBench: 10_000,
+  noThreshold: false,
+  triggerGC: false,
+  setupFiles: [],
+  skip: false,
+  only: false,
+  threshold: 2,
+  convergence: "linear",
+  averageCalculation: "simple",
```

Comment:
is this the right default?

Reply:
Yes I tried to not break compatibility with current version. Once tested will change the defaults in a breaking release.